tests/testthat/test-z-integration.R

```diff
@@ -149,6 +149,7 @@ test_that("git error returns valid json", {
 
 
 test_that("run report honours timeout", {
+  skip("never exits")
```

Comment:
What's going on with this? This because rrq timeout will mean this orderly.server timeout is always skipped? Probably fine as is though as we are going to remove the timeout from orderly.server soon.

Reply:
oh that's probably an omission